From d4fec31889ad660a58dab633c511221feb66e817 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 5 Oct 2013 15:55:06 +0200
Subject: [PATCH 3/3] networking/libiproute: use <linux/if_packet.h> instead of
 <net/if_packet.h>

The musl C library doesn't provide the <net/if_packet.h> since the
corresponding kernel headers <linux/if_packet.h> already provides the
necessary definitions. Replacing <net/if_packet.h> by
<linux/if_packet.h> also removes the need to include
<netpacket/packet.h>

This commit fixes the build of iplink with the musl C library.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 networking/libiproute/iplink.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/networking/libiproute/iplink.c b/networking/libiproute/iplink.c
index bad2017..32ccb1c 100644
--- a/networking/libiproute/iplink.c
+++ b/networking/libiproute/iplink.c
@@ -5,8 +5,7 @@
  * Licensed under GPLv2 or later, see file LICENSE in this source tree.
  */
 #include <net/if.h>
-#include <net/if_packet.h>
-#include <netpacket/packet.h>
+#include <linux/if_packet.h>
 #include <netinet/if_ether.h>
 
 #include "ip_common.h"  /* #include "libbb.h" is inside */
-- 
1.8.1.2

